### DISCUSSION TITLE
Setting every pixel regardless of whether it was already correct?

### DISCUSSION THREAD

**Discussion by Swithord** [rdeepak2002/reddit-place-script-2022#235] [Q&A]
Perhaps this is only a thing that happens on the first run, but my accounts seem to be going through every pixel in the image one by one and setting them to the corresponding colour, irrespective of whether the pixel there was already correct. Is this working as intended, or is something wrong on my part?

**Comment by funwithtriangles** [comment]
This isn't the expected behaviour. Related: https://github.com/rdeepak2002/reddit-place-script-2022/issues/217
